executable/AndroidManifest.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: android_manifest
+  title: Android Binary manifest xml
+  file-extension: xml
+  endian: le
+doc: |
+    Android manifest has its own binary xml format.
+    It is not documented.
```

Comment:
Please (1) fix formatting — it should be 2 space indent at this level, not 4-space indent.

(2) add more information about this format: what is "Android"? what exactly is a "manifest"? this file is placed into "executable" category — so it's something executable? what information does it use, how it is all connected, etc?